python/lsst/sims/photUtils/PhotometricParameters.py

```diff
@@ -45,7 +75,7 @@ def makeDict(value,
     nexp = makeDict(nexpN)
 
     # effective area in cm^2
-    effareaCm2 = numpy.pi * (6.423/2.*100)**2
+    effareaCm2 = np.pi * (6.423/2.*100)**2
```

Comment:
We have the effective diameter of the mirror hidden in a value like this? 
(6.423)?
We should not be doing that.

Reply:
ehhh .. looking at the full class I see lots of other magic numbers. I would suggest just making a single line effDiam = 6.423. #meters and then proceeding as is.